Tests/SwiftStdlibTests/CaseIterableExtensionsTests.swift

```diff
@@ -0,0 +1,51 @@
+// CaseIterableExtensionsTests.swift - Copyright 2023 SwifterSwift
+
+@testable import SwifterSwift
+import XCTest
+
+final class CaseIterableExtensionsTests: XCTestCase {
+    
+    private enum Test: CaseIterable {
```

Comment:
Can you perform this without making the `enum` conform to `Equatable`?
```suggestion
    private enum Test: CaseIterable, Equatable {
```